packages/core/src/agent/openai/worker.ts

```diff
@@ -58,9 +58,9 @@ async function callFunction(
 
   return [
     {
-      content: String(output),
+      content: `${output}`,
       role: "tool",
-      additionalKwargs: {
```

Comment:
rename `kwargs` to `options`. It looks like the way of python. And I don't think there should be compatibility on this, instead should use some JS standard